README.md

```diff
@@ -25,8 +25,6 @@ from coqlspclient.coq_structs import TermType
 
 # Open Coq file
 with CoqFile(os.path.join(os.getcwd(), "examples/example.v")) as coq_file:
-    # Print AST
-    print(coq_file.ast)
```

Comment:
Why was this removed?

Reply:
CoqFile no longer has the ast attribute, it only has steps where each Step has its own AST

Reply:
Oh ye true!